src/components/AddBook/AddBook.module.css

```diff
@@ -30,30 +46,36 @@
 }
 
 .form-container form input {
+  border-radius: 0.3rem;
+  border: solid 2px #e8e8e8;
```

Comment:
- [ ] It will be better if you can create class for repeated rules.For example, in this file you have repeated `solid 2px #e8e8e8` three times, you could have create a class for it and use that  class in HTML elements to keep your CSS more organised.

Reply:
Sure I will implement them. Thanks Mr. @sinansevgi for the info.